File: plugins/devkit/devkit-java-tests/testData/inspections/useHtmlChunkToolTip/SetToolTipTextWithStringFix_after.java
import com.intellij.ide.HelpTooltipKt;
import com.intellij.openapi.util.text.HtmlChunk;

import javax.swing.*;

class SetToolTipTextWithStringFix {
  void test(JComponent comp) {
    HelpTooltipKt.setToolTipText(comp, HtmlChunk.text("hello"));
  }
}

File: plugins/devkit/devkit-kotlin-tests/testData/inspections/useHtmlChunkToolTip/SetToolTipTextWithStringFix_after.kt
@file:Suppress("MISSING_DEPENDENCY_SUPERCLASS") // mockSDK misses some superclasses

import com.intellij.ide.setToolTipText
import com.intellij.openapi.util.text.HtmlChunk
import javax.swing.JComponent

class SetToolTipTextWithStringFix {
  fun test(comp: JComponent) {
      comp.setToolTipText(HtmlChunk.text("hello"))
  }
}
